Code-4-Community/speak-for-the-trees-frontend
# create login/signup pages

## Why

[Monday.com Ticket](https://code4community-team.monday.com/boards/702429655/pulses/788451685)

## This PR
Created login and signup pages (and corresponding containers)
- Look consistent with designs
  - Design uses font-family: Circular, used sans-serif as the fallback (in screenshots)
- Functional links to login/signup pages (no forgot password form exists)
  - Added paths to App
- Use AntD 24 Grid System for layout flexibility/consistency

## Screenshots
Left: Design, Right: Finished Page (excluding NavBar)
<img width="900" src="https://user-images.githubusercontent.com/22990100/97641834-34aa9b00-1a1a-11eb-8054-f83f45f11e1c.png">

## Verification Steps
- Compared side-by-side to see if the pages generally matched the design
- Shrunk/maximized window size to check flexibility
- Clicked links to check functionality
- Checked that color codes matched the design
